code/src/java/pcgen/persistence/lst/URIEntry.java

```diff
@@ -196,10 +189,10 @@ public URIEntry getRelatedTarget(String fileName)
 	 *            The offset from the root URI for the URIEntry
 	 * @return A new URIEntry from the given Campaign name, rootURI and offset
 	 */
-	public static URIEntry getURIEntry(String campaignName, URI rootURI,
-		String offset)
+	static URIEntry getURIEntry(String campaignName, URI rootURI,
+								String offset)
 	{
-		if (offset == null || offset.isEmpty())
+		if ((offset == null) || offset.isEmpty())
```

Comment:
Why does this use both == null and .isEmpty instead of the .isEmpty from earlier? Seems like an inconsistent standard here.